sdk/resourcemanager/azure-resourcemanager-compute/src/test/java/com/azure/resourcemanager/compute/VirtualMachineOperationsTests.java

```diff
@@ -1562,7 +1563,48 @@ public void canSwapOSDiskWithManagedDisk() {
         Assertions.assertNull(vm1.storageProfile().osDisk().managedDisk().diskEncryptionSet());
     }
 
+    @Test
+    public void canCRUDTrustedLaunchVM() {
+        VirtualMachine vm = computeManager.virtualMachines()
+            .define(vmName)
+            .withRegion(Region.US_WEST3)
+            .withNewResourceGroup(rgName)
+            .withNewPrimaryNetwork("10.0.0.0/28")
+            .withPrimaryPrivateIPAddressDynamic()
+            .withoutPrimaryPublicIPAddress()
+            .withPopularLinuxImage(KnownLinuxVirtualMachineImage.UBUNTU_SERVER_20_04_LTS_GEN2)
+            .withRootUsername("Foo12")
+            .withSsh(sshPublicKey())
+            .withTrustedLaunch()
+            .withSecureBoot()
+            .withVTpm()
+            .withSize(VirtualMachineSizeTypes.STANDARD_DS1_V2)
+            .withPrimaryNetworkInterfaceDeleteOptions(DeleteOptions.DELETE)
+            .create();
+
+        Assertions.assertEquals(SecurityTypes.TRUSTED_LAUNCH, vm.securityType());
+        Assertions.assertTrue(vm.isSecureBootEnabled());
+        Assertions.assertTrue(vm.isVTpmEnabled());
+
+        vm.update()
+            .withoutSecureBoot()
+            .withoutVTpm()
+            .applyAsync()
+            // security features changes need restart to take effect
+            .flatMap(VirtualMachine::restartAsync)
+            .block();
```

Comment:
why not use `vm.update()....apply()` instead of `vm.update()....block()`?

Reply:
Need to restart VM after applying the update(`flatMap(VirtualMachine::restartAsync)`), hence chain the operation together and block at last.